cPRONTO/oppni.py

```diff
@@ -1870,7 +1942,7 @@ def submit_jobs():
         if options.run_locally is True and (status_sp is False or status_sp is None):
             raise Exception('Spatial normalization - steps 2 and later failed.')
 
-    # group mask
+    #print(proc_status.gmask)
```

Comment:
remove or change to print("group mask processing status: ".format*proc_status.gmask))